app/CurrikiGo/Canvas/Commands/CreateModuleItemCommand.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\CurrikiGo\Canvas\Commands;
+
+use App\CurrikiGo\Canvas\Contracts\Command;
+
+class CreateModuleItemCommand implements Command
+{
+    public $apiURL;
+    public $accessToken;
+    public $httpClient;
+    private $courseId;
+    private $moduleId;
+    private $itemData;
+    const MODULE_TYPE = 'ExternalTool';
```

Comment:
Same here. Also it would be great if you can add some comment for each variable.